src/ipfs-npm/registry/clone/index.js

```diff
@@ -19,39 +18,87 @@ let latestSeq = 'unknown'
 // const GLOBAL_INDEX = '-/index.json'
 // const NOT_FOUND = '-/404.json'
 
-module.exports = RegistryClone
-
-function RegistryClone (ipfs, opts) {
-  if (!(this instanceof RegistryClone)) {
-    return new RegistryClone(ipfs, opts)
+module.exports = function registryClone (opts) {
+  opts = opts || {
+    store: require('ipfs-blob-store')
   }
 
-  let bsConfig = config.blobStore
-  opts = opts || {}
+  let storeConfig = config.blobStore
 
   if (opts.url) {
     const parsed = multiaddr(opts.url).nodeAddress()
-    bsConfig = Object.assign(bsConfig, {
+    storeConfig = Object.assign(storeConfig, {
       host: parsed.address,
-      port: parsed.port
+      port: parseInt(parsed.port, 10)
     })
   }
 
   if (typeof opts.flush === 'boolean') {
-    bsConfig.flush = opts.flush
+    storeConfig.flush = opts.flush
+  }
+
+  const followConf = {
+    seqFile: config.seqFile,
+    handler: changeHandler,
+    registry: config.registry,
+    skim: config.skim
   }
 
   if (opts.seqNumber) {
-    // TODO: what to do?
+    followConf.since = opts.seqNumber
   }
```

Comment:
@diasdavid `since` is the name `follow-registry` understands, passing `seqNumber` wouldn't work